src/memory-usage.ts

```diff
@@ -65,6 +65,8 @@ class MemoryUsageFeature implements vscodelc.StaticFeature {
     this.context.subscriptions.push(vscode.commands.registerCommand(
         'clangd.memoryUsage.close', () => adapter.root = null));
   }
+  fillInitializeParams?: (params: vscodelc.InitializeParams) => void;
```

Comment:
this declaration is really confusing, did you mean `fillInitializeParams: (params: vscodelc.InitializeParams) {}`?